fv3core/stencils/c_sw.py

```diff
@@ -218,12 +218,12 @@ def divergence_corner(
         with horizontal(region[i_start, :], region[i_end + 1, :]):
             vf = v * dxc * 0.5 * (sin_sg3[-1, 0] + sin_sg1)
 
-        divg_d = vf[0, -1, 0] - vf + uf[-1, 0, 0] - uf
+        divg_d = (vf[0, -1, 0] - vf + uf[-1, 0, 0] - uf) * rarea_c
         with horizontal(region[i_start, j_start], region[i_end + 1, j_start]):
-            divg_d -= vf[0, -1, 0]
+            divg_d = (-vf + uf[-1, 0, 0] - uf) * rarea_c
+
         with horizontal(region[i_end + 1, j_end + 1], region[i_start, j_end + 1]):
-            divg_d += vf
-        divg_d *= rarea_c
```

Comment:
This is causing validation to fail because the math is changed. Excellent. We can justify increasing the max_error slightly in the DivergenceCorner error -- please verify all the errors are tiny (e.g. run the test locally with --print_failures). The one jenkins reports is 3.828570515762384e-07, reference: 3.828570515762432e-07, diff: 4.817499887508833e-21, so actually very close to the default relative error metric of 1e-14.

Reply:
Oh, I see the problem: `rarea_c` needs to be shifted.